src/query/service/src/pipelines/processors/transforms/aggregator/transform_aggregate_partial.rs

```diff
@@ -295,7 +302,8 @@ impl<Method: HashMethodBounds> AccumulatingTransform for TransformPartialAggrega
         if Method::SUPPORT_PARTITIONED {
             if matches!(&self.hash_table, HashTable::HashTable(cell)
                 if cell.len() >= self.settings.convert_threshold ||
-                    cell.allocated_bytes() >= self.settings.spilling_bytes_threshold_per_proc
+                    cell.allocated_bytes() >= self.settings.spilling_bytes_threshold_per_proc ||
```

Comment:
`spilling_bytes_threshold_per_proc` default is 0, so the check:
```
cell.allocated_bytes() >= self.settings.spilling_bytes_threshold_per_proc
```
always true?

Reply:
We set spilling_bytes_threshold_per_proc = Usize::Max if spilling_bytes_threshold_per_proc == 0;